src/bootstrap/doc.rs

```diff
@@ -263,9 +258,7 @@ fn invoke_rustdoc(builder: &Builder, target: Interned<String>, markdown: &str) {
         t!(t!(File::create(&version_info)).write_all(info.as_bytes()));
     }
 
-    let mut cmd = Command::new(&rustdoc);
-
-    builder.add_rustc_lib_path(compiler, &mut cmd);
+    let mut cmd = builder.rustdoc_cmd(builder.compiler(0, build.build));
```

Comment:
The `0` here didn't change, but is somewhat suspicious. I think we may want to request a stage from the caller, who would get it from `builder.top_stage` in most cases, probably.